### ISSUE TITLE
[Accessibility] Add aria-controls

### ISSUE THREAD

**Issue by damian-krupa** [sagalbot/vue-select#1798]
I am using an automated accessibility testing tool called Level Access Assistant. It shows an issue for Select.vue component - aria-controls attribute is lacking.

**Describe the solution you'd like**
add this line of code :aria-controls="`vs${uid}__listbox`" in the Select.vue combobox div, right under `aria-owns`

**Additional context**
Error message: This DIV (role=combobox) must have all of the following attributes: aria-controls | aria-expanded

**Comment by damian-krupa** [comment]
I have opened a PR some time ago to fix this. Please take a look, it's a one-liner https://github.com/sagalbot/vue-select/pull/1802

**Comment by gender404** [comment]
That looks like the same bug that I'm running into with my audit.
